Automattic/wp-calypso
# Customer Home: Adjust the eligibility checks to add email to include Professional Email

#### Changes proposed in this Pull Request

* Update to the email eligibility checks in the quick links to include Professional Email  eligibility. We were currently only checking for Google Workspace suitability.

#### Testing instructions



* Run this branch locally or via the [live branch](https://calypso.live/?branch=update/include-titan-checks-in-add-email-links-for-custom-home)..
* Ensure that you have sites with following scenarios exclusively:
    1) A site with a free sub-domain
    2) A site with a registered domain with no email subscription and no domain restrictions, nor dispute flags
    3) A site with a mapped domain with no email subscription
    4) A site with a sub-domain
    5) A site with domain with an unverified domain, or dispute flags
    6) A site with a domain with an existing email subscription ( Google Workspace, Professional Email )

* Confirm that the <kbd>Add email</kbd> link only appears for scenario No. 2 and 3 above

<img width="1068" alt="Screenshot 2021-06-18 at 10 36 56 AM" src="https://user-images.githubusercontent.com/277661/122541317-68761780-d021-11eb-93b8-4c6d5ec67558.png">
